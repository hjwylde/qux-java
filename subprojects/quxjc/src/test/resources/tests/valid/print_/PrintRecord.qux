package valid.print_

void main([str] args):
    print {x: true, y: null, z: -5}
    print {x: null, y: null}
    print {x: {x: null}, y: [{x: 5}]}

