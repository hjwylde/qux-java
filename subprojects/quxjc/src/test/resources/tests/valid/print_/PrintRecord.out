{x: true, y: null, z: -5}
{x: null, y: null}
{x: {x: null}, y: [{x: 5}]}
